x/executionlayer/util.go

```diff
@@ -39,3 +40,15 @@ func toBytes(keyType string, key string,
 	}
 	return bytes, nil
 }
+
+func isEmptyBytes(src []byte) bool {
```

Comment:
Can we just use `len(src) == 0`?

Reply:
https://github.com/hdac-io/friday/pull/64/commits/8cf06f83e0cb1e2957d37787d33017d767ac2ba5